InvoiceMaker/InvoiceMaker/Controllers/WorkDoneController.cs

```diff
@@ -0,0 +1,78 @@
+﻿using InvoiceMaker.Data;
+using InvoiceMaker.FormModels;
+using InvoiceMaker.Models;
+using InvoiceMaker.Models.Repositories;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+
+namespace InvoiceMaker.Controllers
+{
+    public class WorkDoneController : Controller
+    {
+        // GET: WorkDone
+        private Context context;
+
+        public WorkDoneController()
+        {
+            context = new Context();
+        }
+        [HttpGet]
+        public ActionResult Create()
+        {
+            CreateWorkDone model = new CreateWorkDone();
+            model.PopulateSelectLists();
+            //model.Clients = new ClientRepository(null).GetClients();
+            model.WorkTypes = new WorkTypeRepo(context).GetWorkTypes();
```

Comment:
Looks like you're retrieving work types within the `PopulateSelectLists` method, so you can remove this line of code.